features/step.nested_folders.feature

```diff
@@ -0,0 +1,14 @@
+Feature: Identically named folders and modules in nested directories are all discovered
+
+Scenario: Steps are found even in identically-named folders, with differing or identical module names
+    When a step is implemented in the root folder
+    And a step is implemented in a sub folder
+    And a step is implemented in a sub-sub folder
+    And a step is implemented in another sub folder
+    And a step is implemented in another sub-sub folder with an identical name
+    And a step module with an identical name exists in the root folder
+    And a step module with an identical name exists in a sub folder
+    And a step module with an identical name exists in a sub-sub folder
+    And a step are collected recursively from nested folders under sub-folder3
+    And a step with identical names from different levels in sub-folder3 should be executed successfully
```

Comment:
I guess you intend to specify this:

```suggestion
    And a step is implemented in a sub-sub-sub folder with no module in the intermediate level
    And a step module with an identical name exists in the sub-folder two levels deeper
```

The way you have them currently worded they're more like _Then_ steps, but the step implementation shows that they are both in fact _When_ steps (there is no `assert`).

Reply:
Hi, done.